vocabulary-draft/continuous-deployment-pipeline-events.md

```diff
@@ -0,0 +1,26 @@
+# Continuous Deployment Pipelines Events 
+
+__Note:__ This is a work-in-progress draft version and is being worked on by members of the Events SIG. You are very welcome to join the work and the discussions!
+
+These events are related to continous deployment pipelines and their target environments. 
+These events can be emitted by environments to report where software artifacts such as services, binaries, deamons, jobs or embedded software are running. 
+
+The term Service is used to represent a running Artifact. This service can represent a binary that is running, a deamon, an application, a docker container, etc.
+The term Environment represent any platform which has all the means to run a Service. 
+
+- **Environment Created**: an environment has been created and it can be used to deploy Services
+- **Environment Modified**: an environment has been modified, this event advertise the changes made in the environment
+- **Environment Deleted**: an environment has been deleted and cannot longer be used
```

Comment:
I can see a need for the definition of environments during the test phase. Do we intend to create environment events in the CI domain also or should the test be able to use these events also?

The background for the comment is that for us it is more interesting to trace the environment during the test than it is to track it when something has been deployed.

Reply:
@m-linner-ericsson I can see the point that you are trying to make, but I think that we need to cover both. We can say that Environments will be created or provisioned for tests to run, which is in fact pretty similar in nature to create an environment where the software will run. But  I might not be fully understanding your suggestion here, are you proposing to create some other kind of events for test environments?